mmv1/templates/terraform/examples/subnetwork_reserved_internal_range.tf.erb

```diff
@@ -0,0 +1,22 @@
+resource "google_compute_subnetwork" "subnetwork_with_reserved_internal_range" {
+  name                    = "<%= ctx[:vars]['subnetwork_name'] %>"
```

Comment:
You'll need to add `provider = google-beta` to these resources.

Reply:
@trodge added!